app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -51,7 +51,7 @@ The global control plane on Kubernetes must reside on its own Kubernetes cluster
 1.  Set the `controlPlane.mode` value to `global` in the chart (`values.yaml`), then install. On the command line, run:
 
     ```sh
-    helm install {{ site.mesh_helm_install_name }} --create-namespace --namespace {{site.mesh_namespace}} --set controlPlane.mode=global {{ site.mesh_helm_repo }}
+    helm install {{ site.mesh_helm_install_name }} --create-namespace --namespace {{site.mesh_namespace}} --set {{set_flag_values_prefix}}controlPlane.mode=global {{ site.mesh_helm_repo }}
```

Comment:
should it be `site.set_flag_values_prefix` also please prefix these with `mesh_`

Reply:
Ha nevermind then. I didn't realise this alreayd existed!